homeassistant/components/wemo/light.py

```diff
@@ -133,27 +126,36 @@ def brightness(self) -> int:
         return cast(int, self.light.state.get("level", 255))
 
     @property
-    def hs_color(self) -> tuple[float, float] | None:
-        """Return the hs color values of this light."""
-        if xy_color := self.light.state.get("color_xy"):
-            return color_util.color_xy_to_hs(*xy_color)
-        return None
+    def xy_color(self) -> tuple[float, float] | None:
+        """Return the xy color value [float, float]."""
+        return self.light.state.get("color_xy")  # type:ignore[no-any-return]
 
     @property
     def color_temp(self) -> int | None:
         """Return the color temperature of this light in mireds."""
         return cast(Optional[int], self.light.state.get("temperature_mireds"))
 
+    @property
+    def color_mode(self) -> ColorMode:
+        """Return the color mode of the light."""
+        if "colorcontrol" in self.light.capabilities:
+            return ColorMode.XY
+        if "colortemperature" in self.light.capabilities:
+            return ColorMode.COLOR_TEMP
```

Comment:
This implementation now assumes there is no light which has both `"colorcontrol"` and `"colortemperature"`, it would mean there's no wemo color light which also supports white adjustable color temperature.
@esev can you confirm that's the case?

Reply:
I suspect this may not be the case. https://www.belkin.com/us/support-article?articleNum=156305. Can the PR be modified to support lights that support both?

Okay if we wait a few more days on this PR? I've ordered one of these to test with. It should arrive on Thursday.

Reply:
@esev Sure, we can wait a bit longer! If you prefer and you have the time, please go ahead and open your own PR implementing color mode support or just suggest changes to this one 👍

What's left to do is basically to confirm if any bulbs support both color and color temperature, and if they do make sure the `color_mode` property correctly reflects the state of the light.